modules/react/collection/lib/ListBox.tsx

```diff
@@ -4,21 +4,21 @@ import {
   createElemPropsHook,
   createSubcomponent,
 } from '@workday/canvas-kit-react/common';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {Box, Flex, FlexProps} from '@workday/canvas-kit-react/layout';
 
 import {useListModel} from './useListModel';
 import {useListRenderItems} from './useListRenderItem';
 import {useListItemRegister} from './useListItemRegister';
 
-export interface ListBoxProps extends Omit<BoxProps, 'children'> {
-  children?: React.ReactNode | ((item: any) => React.ReactNode);
+export interface ListBoxProps<T = any> extends Omit<FlexProps, 'children'> {
+  children?: React.ReactNode | ((item: T, index: number) => React.ReactNode);
 }
 
 export const ListBoxItem = createSubcomponent('li')({
   displayName: 'Item',
   modelHook: useListModel,
   elemPropsHook: useListItemRegister,
-})<BoxProps>((elemProps, Element) => {
+})<FlexProps>((elemProps, Element) => {
```

Comment:
```suggestion
})<ExtractProps<typeof Flex>>((elemProps, Element) => {
```

Reply:
You forget the `never`:
```suggestion
})<ExtractProps<typeof Flex, never>>((elemProps, Element) => {
```